app/models/web_hook_event_type.rb

```diff
@@ -26,13 +82,13 @@ class WebHookEventType < ActiveRecord::Base
   def self.active
     ids_to_exclude = []
     unless defined?(SiteSetting.solved_enabled) && SiteSetting.solved_enabled
```

Comment:
All these things are bleh (not your fault) -- we should not be referring to plugins like this in core, but rather having a plugin API to register them. Maybe make a internal TODO for this?

Reply:
Agree, I will create a TODO after this PR is merged (I don't want someone to pick it up before this change is finished)

Reply:
@lis2 I think you could do this now if you haven't yet, this PR should be resolved soon :)